docs/content/troubleshooting/troubleshooting-with-app-protect-dos.md

```diff
@@ -37,11 +37,11 @@ For App Protect Dos specific logs, look for messages starting with `APP_PROTECT_
 
 ### Check events of an Ingress Resource
 
-Follow the steps of [Checking the Events of an Ingress Resource](/troubleshooting/#checking-the-events-of-an-ingress-resource).
+Follow the steps of [Checking the Events of an Ingress Resource]({{< relref "troubleshooting/troubleshoot-ingress-controller.md#checking-the-events-of-an-ingress-resource" >}}).
```

Comment:
what does `relref` do?

Reply:
It's a hugo command that returns the relative path to a document:
https://gohugo.io/functions/relref/

Provides more flexibility and features than a regular markdown link and is our preferred method for intra-documentation links in the docs.nginx.com